installhelper.py

```diff
@@ -46,20 +47,27 @@ def make_semver(version_str):
     return version
 
 
-def update_version():
-    with open(f"{os.path.dirname(__file__)}/chia-blockchain-gui/package.json") as f:
-        data = json.load(f)
-
+def get_chia_version():
```

Comment:
would you mind adding a type hint for the return value here? I would imagine it's: ` -> str`

Reply:
It should be. Thank you!